src/main/java/com/google/devtools/build/lib/query2/output/CqueryOptions.java

```diff
@@ -43,9 +43,8 @@ public TransitionsConverter() {
       effectTags = {OptionEffectTag.TERMINAL_OUTPUT},
       help =
           "The format in which the cquery results should be printed. Allowed values for cquery "
-              + "are: label, transitions, proto. If you select 'transitions', you also have to "
-              + "specify the --transitions=(lite|full) option."
-  )
+              + "are: label, transitions, proto, textproto. If you select 'transitions', you also "
```

Comment:
Can you please also document in AqueryOptions the valid values for --output?

Reply:
Done.